.buildkite/pipelines/security_solution/base.yml

```diff
@@ -0,0 +1,11 @@
+steps:
+  - command: .buildkite/scripts/pipelines/security_solution_quality_gate/mki_security_solution_cypress.sh
+    label: 'Serverless Security Cypress Tests'
+    agents:
+      queue: n2-4-spot
+    timeout_in_minutes: 300
+    parallelism: 6
```

Comment:
Why 6 as parallelism?

Reply:
I split the test suites to non explore/investigations and these two categories on their own.